README.md

```diff
@@ -9,3 +9,11 @@ Currently supported formats
 * STL
 
 We plan to add support for Wavefront OBJ format next.
+
+## Documentation
+
+* [Documentation]
+* [Coverage]
```

Comment:
@9prady9 made these changes, so that we can look at coverage or documentation and get to see coverage limitations visually without needing to rely coveralls.

Reply:
Can you please enable github pages for this project.